<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewCourses</class>
 <widget class="QDialog" name="ViewCourses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="filterBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="filterSpecific">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter by</string>
   </property>
  </widget>
  <widget class="QListWidget" name="coursesView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
